extra-requirements.txt

```diff
@@ -72,6 +72,7 @@ jsonschema:                 cicd
 dgl:                        cicd
 black==20.8b1:              test
 matplotlib:                 devel
+portforward>=0.2.4:         devel
```

Comment:
This would be ideal @deepankarm but we have a problem about the hub images, there is currently no way to force them to have a different set of dependencies